packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -38,18 +40,35 @@ class TimelineScreenBody extends StatefulWidget {
 }
 
 class TimelineScreenBodyState extends State<TimelineScreenBody> {
-  @visibleForTesting
-  final controller = TimelineController();
+  TimelineController controller;
+
+  StreamSubscription selectedFrameSubscription;
 
   @override
   void initState() {
```

Comment:
This override can be removed.

Reply:
Done.

Reply:
@override is needed w/o

  info - Annotate overridden members at lib/src/timeline/flutter/flutter_frames_chart.dart:104:8 - (annotate_overrides)

Reply:
I mean the method can be removed.  It's a no-op right now.

Reply:
Ok, sorry took it literally.